pyproject.toml

```diff
@@ -27,16 +27,29 @@ readme = "README.md"
 simplesecurity = 'simplesecurity:cli'
 
 [tool.poetry.dependencies]
-python = "^3.7"
-poetry = { version = "<2,>=1.1.10", optional = true }
-bandit = { version = "<2,>=1.7.0", optional = true }
-safety = { version = "<2,>=1.10.3", optional = true }
-dodgy = { version = "<2,>=0.2.1", optional = true }
-dlint = { version = "<2,>=0.11.0", optional = true }
-semgrep = { version = "<2,>=0.67.0", optional = true }
+python = "^3.8"
+poetry = "^1.1.10"
```

Comment:
If using extras, can you mark these as optional please?

Reply:
I'm referring to all deps (can't make python optional). GH is a nightmare in conversation view (might be worth going to https://github.com/FHPythonUtils/SimpleSecurity/pull/6/files, to see the full diff)

Reply:
Found it! Thanks for the feedback!